archetypes/adr.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+type: "adrs"
+date: "{{ time.Now.Format "2006-01-02" }}"
+decision: "Decision summary goes here."
+status: "proposed"
```

Comment:
I am wondering if we should have a `draft` status. Do you see value in adding that?

@PGaigole, What are the different values for status as per you?

Reply:
@hussainweb I had three values in mind. Accepted, Proposed and Declined.
I did see the documentation had a 4th value as Deprecated

Reply:
Hmm... I think draft might be useful in case the ADR is still not ready for review. But we can support that later. Setting it to "proposed" by default is good for now.